package.json

```diff
@@ -17,7 +23,16 @@
     "url": "https://github.com/kagisearch/browser_extensions/issues"
   },
   "homepage": "https://github.com/kagisearch/browser_extensions#readme",
-  "dependencies": {
-    "adm-zip": "0.5.9"
+  "devDependencies": {
+    "adm-zip": "0.5.10",
```

Comment:
I updated this dependency, to this latest version which has been around for over 6 months.